diff --git a/Numix-Circle/index.theme b/Numix-Circle/index.theme
index ff56cc0..548805c 100644
--- a/Numix-Circle/index.theme
+++ b/Numix-Circle/index.theme
@@ -1,8 +1,8 @@
 [Icon Theme]
-Name=Numix Circle
-Comment=Numix icon theme with a circular apps icons design
+Name=Korora
+Comment=Korora icon theme that extends on Numix-Circle
 
-Inherits=Numix
+Inherits=korora-base
 
 Example=directory-x-normal
 
